### ISSUE TITLE
Updating layers properties don't update the map

### ISSUE THREAD

**Issue by fsteinmetz** [jupyter-widgets/ipyleaflet#270]
I am creating a map with some layers:
```python
from ipyleaflet import Map, TileLayer, Marker, __version___
# __version__ is '0.9.2'
m = Map()
marker = Marker()
m.add_layer(marker)
lay = TileLayer()
m.add_layer(lay)
m
```

Shouldn't any further update of layers trigger an update of the map ? It does for the map but not for the layers, for example:

```python
marker.title='foo'
lay.attribution = 'foo'
lay.url='http://toolserver.org/tiles/hikebike/{z}/{x}/{y}.png'
```

**Comment by martinRenou** [comment]
Yes, that's a bug. Changing the URL of the `TileLayer` doesn't do anything... It should not be too difficult to fix. In the meantime, I suggest you create a new `TileLayer` with the URL you want and add it to the map (you can even remove the old one before adding the new one).
You you want to contribute to ipyleaflet by fixing this issue, feel free to do it :)

**Comment by fsteinmetz** [comment]
Thanks for the confirmation and for the tip ;)
I'm still far from being able to contribute, but I'll think about it, because this project looks very promising and there are certainly great thinks to be done.

**Comment by martinRenou** [comment]
Thanks for reporting :) I'll try to fix it then

**Comment by martinRenou** [comment]
Updating the URL is fixed on my branch #271. However, changing the `attribution` will be a bit more difficult, there is no clean solution that I can think of, so I guess we can keep this issue open for now.

**Comment by fsteinmetz** [comment]
Thanks Martin :)
The `attribution` and `marker.title` were just tests, I was primarily interested in the update of the `TileLayer`'s url. So I don't mind if you close the issue.
